assets/js/localization.js

```diff
@@ -212,13 +225,29 @@ function getLocales() {
             return a[1].toLowerCase().localeCompare(b[1].toLowerCase());
         });
         $('.localeSelect').empty();
+        $('#localeDropdown').empty();
         $.each(localePairs, function () {
+            var isRtlLanguage = (rtlLanguages.indexOf(this[0]) !== -1);
             $('.localeSelect').append(
-                $('<option></option>')
+                $('<option>')
                     .val(this[0])
                     .text(this[1])
-                    .prop('dir', rtlLanguages.indexOf(this[0]) !== -1 ? 'rtl' : 'ltr')
+                    .prop('dir', isRtlLanguage ? 'rtl' : 'ltr')
             );
+
+            $('#localeDropdown').append(
+                $('<li></li>').append(
+                    $('<a>')
+                        .data('locale', this[0])
+                        .text(this[1])
+                        .prop('dir', isRtlLanguage ? 'rtl' : 'ltr')
+                        .css({
```

Comment:
Actually, on second thought, this should all go in the stylesheet. All you need to do is `#localeDropdown li[dir='rtl'] { ... } `.